packages/RatPack.package/RPModel.class/class/updateObject.withJson..st

```diff
@@ -0,0 +1,11 @@
+object conversion
+updateObject: aRPModelClassObject withJson: aJsonString
+
+	| newObject aJsonObject|
+	
+	aJsonObject := self createJsonObject: aJsonString. 
+	newObject := aRPModelClassObject. 
+	self dataTypes keysDo: 
+		[:key | newObject instVarNamed: key put: (aJsonObject at: key 
+								ifAbsent: [newObject instVarNamed: key])].
```

Comment:
Why did you chose to iterate this way? I'd iterate over the keys in `aJsonObject`, that would remove the necessity for the `ifAbsent` checks.

Reply:
Done

Reply:
It was done this way cause it was this way in the createObject method. so legacy reasons with not enough refactoring done as it seams ;)